js_modules/dagster-ui/packages/ui-core/src/assets/catalog/AssetCatalogTableV2.tsx

```diff
@@ -424,7 +427,10 @@ const Table = React.memo(
               {loading ? (
                 <Skeleton $width={300} $height={21} />
               ) : (
-                <LaunchAssetExecutionButton scope={scope} />
+                <LaunchAssetExecutionButton
+                  scope={scope}
+                  additionalDropdownOptions={extraDropdownOptions}
```

Comment:
This all looks good! My only feedback is that this non-launch-related option appearing in the dropdown joined to the execute button feels sorta weird.  

I know the dropdown doesn't exist on this page, but it feels like it'd make more sense to include it in a separate dropdown like this:  (from the Catalog page)

![image](https://github.com/user-attachments/assets/3a8cc2ed-a044-493d-9473-0ac52f1199c3)